routes/api/v1/signup.js

```diff
@@ -0,0 +1,18 @@
+var express = require('express');
+var router = express.Router();
+
+router.post('/', (req, res, next) => {
+  var user = req.body
+  var db = req.app.get('db')
+  if(user.username !== undefined && user.password !== undefined) {
```

Comment:
I am really glad you brought this up. I came across JWT's when researching how to create a session. Thanks for the video. I will watch it and get a working implementation of JWTs.